src/devices/SenseHat/SenseHat.cs

```diff
@@ -133,23 +147,32 @@ public SenseHat()
         /// <inheritdoc/>
         public void Dispose()
         {
-            _ledMatrix?.Dispose();
-            _ledMatrix = null!;
+            if (_shouldDispose)
+            {
+                _i2cBus?.Dispose();
```

Comment:
note we don't need to dispose of devices separately here since this will be handled by bus